osv/ecosystems/bioconductor.py

```diff
@@ -57,6 +57,9 @@ def _enumerate_versions(self,
     """Helper method to enumerate versions from a specific URL."""
 
     versions = []
+    # Currently breaking on 3.19,
+    # see https://github.com/google/osv.dev/pull/1477/files#r1575458933
+    bioc_versions = bioc_versions[1:]
```

Comment:
Rather than assume [0] is 3.19, can we just do a bioc_versions.remove() ?

Reply:
Addressed in 7e9644a